readme.md

```diff
@@ -44,7 +44,7 @@ Yields:
 
 ## API
 
-### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type\], callback\[, reverse\])
+### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type], callback\[, reverse])
```

Comment:
I actually think it’s pretty nifty that it only does one, I remember when I turned automatic escaping on in remark itself it removed lots of paired escapes.

Maybe instead of your suggestion we should search for new ways to remove the first escape too? E.g., Only escaping the left backet if it could actually turn into a link (`](`), or a reference? `][`.

Anyway, it’s a problem for sure, the cases where an escape is “false positive” are indeed slightly frustrating...